conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/ApiDataHelpersTest.scala

```diff
@@ -23,7 +23,7 @@ class ApiDataHelpersTest extends WordSpec with Matchers {
         encodeAny(java.lang.Integer.valueOf(1)) shouldBe Json.fromInt(1)
         encodeAny(java.lang.Long.valueOf(1L)) shouldBe Json.fromLong(1)
         encodeAny(java.math.BigDecimal.ZERO) shouldBe Json.fromBigDecimal(0)
-        encodeAny(java.sql.Timestamp.valueOf("2020-10-10 08:00:00")) shouldBe Json.fromLong(1602309600000L)
+        encodeAny(new java.sql.Timestamp(0)) shouldBe Json.fromLong(0L)
```

Comment:
Is this actually highlighting that we don't handle timestamps correctly?

Would this change simply but the dirt under the rag?

Reply:
That is valid question. I will think about better way of "fixing" time-zone issue for our tests.